go.mod

```diff
@@ -11,4 +11,4 @@ require (
 	google.golang.org/grpc v1.19.0
 )
 
-go 1.13
+go 1.14
```

Comment:
Nit: can we move this above the `require` statement?

Reply:
done